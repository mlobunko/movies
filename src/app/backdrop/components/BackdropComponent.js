import React from 'react';
import PropTypes from 'prop-types';

import { StyledBackdropComponent } from './styles';

export const BackdropComponent = props => (
  <StyledBackdropComponent {...props} />
);

BackdropComponent.displayName = 'BackdropComponent';

export default BackdropComponent;

BackdropComponent.propTypes = {
  backdropPath: PropTypes.string
};
