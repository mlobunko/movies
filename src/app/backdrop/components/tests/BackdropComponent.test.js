import React from 'react';
import { shallow } from 'enzyme';

import { BackdropComponent } from '../BackdropComponent';

describe('render', () => {
  it('renders correctly', () => {
    const props = {
      backdropPath: '/123.jpg'
    };
    const wrapper = shallow(<BackdropComponent {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
