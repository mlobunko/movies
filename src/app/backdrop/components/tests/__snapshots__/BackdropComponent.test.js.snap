// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render renders correctly 1`] = `
<StyledBackdropComponent
  backdropPath="/123.jpg"
/>
`;
